.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -1,6 +1,6 @@
 name: "\U0001F680 New feature proposal"
 description: Propose a new feature to be added to Vite
-labels: ["enhancement: pending triage"]
+labels: ['enhancement: pending triage']
```

Comment:
If preferred, we can override the singleQuote setting for yml files :thinking:

Reply:
@patak-js I configured json5 and yml to use double quotes and also preserve quotes for keys in json5 to look more like native json